### DISCUSSION TITLE
Document Preview to land on searchTerm page

### DISCUSSION THREAD

**Discussion by AyushSingh1699** [microsoft-search/pnp-modern-search#2179] [Q&A]
I have been using PnP-modern-search webpart. I would like to add a certain functionality where I want to land on the  exact page of the preview document where my highlighted search term appears. The description of my search result pulls the same information irrespective of the page,  so should my preview document.
Any suggestions, references or links for the same would prove to be helpful.

**Comment by wobba** [comment]
There is no support in the the web viewer/edit programs for files to pass in the query terms and then re-search to find the correct page today. And the search index don't contain page reference to the matches today either.
